crates/tabby/src/serve/completions/prompt.rs

```diff
@@ -361,31 +361,27 @@ mod tests {
     #[test]
     fn test_build_prefix_readable() {
         let snippets = vec![
-            "def snippet_1():\n    print(\"This is snipptet 1\")\n".to_string(),
-            "def snippet_2():\n    print(\"This is snipptet 2\")\n".to_string(),
-            "def snippet_3():\n    print(\"This is snipptet 3\")\n".to_string(),
-            "def snippet_4():\n    print(\"This is snipptet 4\")\n".to_string(),
-            "def snippet_5():\n    print(\"This is snipptet 5\")\n".to_string(),
+            "res_1 = invoke_function_1(n)".to_string(),
+            "res_2 = invoke_function_2(n)".to_string(),
+            "res_3 = invoke_function_3(n)".to_string(),
+            "res_4 = invoke_function_4(n)".to_string(),
+            "res_5 = invoke_function_5(n)".to_string(),
         ];
 
         let prefix = "def this_is_prefix():\n";
```

Comment:
before prefix, there should be relevant keyword contains these `invoke function`, thus these function can be searched from index.